PermissionsEx/permissions.yml

```diff
@@ -7277,6 +7277,26 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.lightning
+  electro:
```

Comment:
There are two electros, Jamie Foxx and Francine Frye, which one is this? You will have to update